docs/add-code-flow.md

```diff
@@ -100,4 +96,8 @@ Within the function, a new `Adder` is created with the configured `Blockstore` a
 
   - **[`adder.PinRoot()`](https://github.com/ipfs/go-ipfs/blob/v0.4.18/core/coreunix/add.go#L171)** - *Pin all files under the `MFS` **root***
 
-    The whole process ends with `PinRoot` recursively pinning all the files under the `MFS` **root**
\ No newline at end of file
+    The whole process ends with `PinRoot` recursively pinning all the files under the `MFS` **root**
+
+# Also see
+1. https://github.com/ipfs/go-ipfs/#development
```

Comment:
Sorry, I probably wasn't very clear, I meant for you to add a link in the https://github.com/ipfs/go-ipfs/#development section pointing here, so this document wouldn't get lost in the `docs` directory.

Reply:
Oh you meant only in the issue conversation space and not the actual doc?! Oops!

Reply:
In the actual entry-point README doc, but instead of pointing from here to there, point there to here.

Reply:
Oh dang. I completely misunderstood.  I didn't realize the link was the the top README.md doc. Alright I will go ahead and add it to the README.md

Reply:
Oh, you're right, sorry, I didn't realize the link was copied without the `README.md` path :grimacing: